android/src/main/java/com/appgoalz/rnjwplayer/RNJWPlayerView.java

```diff
@@ -682,7 +682,9 @@ private void doBindService() {
     private void doUnbindService() {
         if (mIsBound) {
             // Detach our existing connection.
-            mActivity.unbindService(mServiceConnection);
+            if(mServiceConnection != null) {
```

Comment:
Aku chybu to tu vyhadzovalo? Pripada mi to ako barlicka na zakrytie skutocneho problemu.

Reply:
podla mna toto david uz fixol